src/ffi/c.rs

```diff
@@ -138,6 +139,18 @@ pub struct Stream<D: Direction> {
     pub _marker: marker::PhantomData<D>,
 }
 
+impl<D: Direction> Stream<D> {
+    pub fn msg(&self) -> Option<&'static str> {
```

Comment:
To confirm, are you sure that all C backends use this as `&'static str` in that it always has a static lifetime?

If not I think the error will need to start containing a `String`?

Reply:
Well zlib does (and miniz just never changes `msg` from NULL), and I figure that if we eventually need to store more dynamic error messages we can change it to Cow. `De/CompressError.message` only returns a `&str`, not a `&'static str`, so I figure this should be fine.